keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -426,18 +427,14 @@ func shuffleDealAndPick(v, nq uint64,
 		minLen = lenI
 		bestIdx = ii
 	}
-	return shuffleDealAndPick(vNext, nq-1, mrNext, nRem-1, minLen, bestIdx)
-}
-
-func lengthOfQueue(i int) int {
-	return i % 10 // hack for this PoC
+	return shuffleDealAndPick(vNext, nq-1, lengthOfQueue, mrNext, nRem-1, minLen, bestIdx)
 }
 
 func main() {
 	numQueues = uint64(128)
 	handSize := 6
 	hashValue := rand.Uint64()
-	queueIndex := shuffleDealAndPick(hashValue, numQueues, func(i int) int { return i }, handSize, math.MaxInt32, -1)
+	queueIndex := shuffleDealAndPick(hashValue, numQueues, func (idx int) int {return idx % 10}, func(i int) int { return i }, handSize, math.MaxInt32, -1)
```

Comment:
I think the function was actually slightly clearer when named with the comment. (just a thought, not asking for a change)